src/operations/skeletons.ts

```diff
@@ -138,7 +138,7 @@ function asAmountV2(amount: AmountType): AmountTypeV2 {
   // convert an AmountType v1 or v2 to an AmountTypeV2.
   // the "units" of a v1 amount type are always 'BTC'
   if (typeof amount === 'number') {
-    return { units: 'BTC', amount: BigInteger.fromByteArrayUnsigned(String(amount)) }
```

Comment:
Note to reviewers:
`bigi`, a BigInteger lib, is a relatively large, no longer maintained dependency that appears to have been originally used in our code due to it being a child dependency, of a child dependency, of an old version of `bitcoinjs-lib`. 

Since then, all of our child dependencies have settled onto the `bn.js` lib. 

This particular function usage `fromByteArrayUnsigned(someNumber.toString())` is misleading. View the src https://github.com/cryptocoinjs/bigi/blob/cb702633d6587a4326cc8c79b4024444caf9531f/lib/convert.js#L12 to see that this usage always falls through to `fromString()`. 

The refactor to using `bn.js` takes this into consideration.

Reply:
Excellent -- yes, that `.fromByteArrayUnsigned` was always an incredibly misleading function.